Migrate Packer script to public repo (#28)
iterative/studio-selfhosted: packer/README.md

> # Studio Selfhosted AMI instance

Reviewer: ```suggestion
# Studio Selfhosted AMI
```

> This AMI contains K3s with preconfigured Ingress Nginx. 

Reviewer: ```suggestion
This AMI contains K3s along with a preconfigured ingress-nginx reverse proxy. 
```

> ## Studio Installation
> To deploy Studio please:
> 1. Run instance with AMI, 
> 2. Open on Security Group ports: 22, 80, 443(if https is needed),
> 3. Login though ssh to the instance,
> 3. Proceed with the instructions of [Studio helm chart](https://github.com/iterative/helm-charts), how tu configure and install the Studio.

Reviewer: ```suggestion
## Installation
1. Create an EC2 instance with the AMI 
2. In the Security Group, allow the following ports: 22, 80, and 443 (if TLS is needed)
3. Log in to the instance with SSH
4. Proceed with the [Studio Helm chart instructions](https://github.com/iterative/helm-charts) on how to configure and install Studio.
```

Author: @jesper7 How about sharing the link to Notion?

Reviewer: Good idea. Let's do it

Author: I have added a link to new docs :-) (I know they are not ready but they will be :P)
